### ISSUE TITLE
[FEATURE] Media Tab

### ISSUE THREAD

**Issue by DeMoN100** [vitorpamplona/amethyst#576]
Like twitter had a tab next to replies called Media that displays every note with a image or video in it only in this tab.

**Comment by vitorpamplona** [comment]
Unfortunately, there isn't a good way to request posts with links inside the content. But we do offer the middle button in the bottom's navigation bar to access Images and Videos that were posted outside of a note (without the textual context)

**Comment by DeMoN100** [comment]
Can't you just add the tab next to replies on a profile page like twitter & only list post there with a photo or video? Most of them are uploaded so the urls loading like how they do on regular notes shouldn't be an issue.  Can't it just copy the wall but only hide what doesn't have a photo or video attached? The middle button you suggested isn't a solution I don't want random profiles media attachments. Like twitter you know the tab on their profile next to replies shows all media they posted on their profile. It's not random like instagrams explore. What is the issue needing resolve?

**Comment by vitorpamplona** [comment]
You can change the middle button's list to be only the people you follow. Just click on the top bar and select all follows.

**Comment by DeMoN100** [comment]
That's still random & not specific to one profile.

**Comment by vitorpamplona** [comment]
ohhh You want a new tab on the profile, not as a feed.

**Comment by DeMoN100** [comment]
Correct that's how twitter did it would like to see that here on peoples profiles as well.

**Comment by vitorpamplona** [comment]
We have had this one for a few versions already. Closing...

**Comment by DeMoN100** [comment]
You must have recently just added it to be responding to this 2 years later saying we had it already. Next time when there's an update that you add something related towards an issue it would be appropriate to comment on the issue the day you add it.

**Comment by vitorpamplona** [comment]
Yes, apologies. We ended up implementing it in a completely unrelated workstream from this issue. That's why it wasn't linked.
